platform-includes/migration/javascript-v8/important-changes/javascript.gatsby.mdx

```diff
@@ -6,6 +6,24 @@ If you need to support older versions of React, please use Sentry Gatsby SDK `7.
 
 <Include name="migration/javascript-v8/compatible-browsers" />
 
+### Updated error types to be `unknown` instead of `Error`.
+
+In v8, we are changing the `ErrorBoundary` error types returned from `onError`, `onReset`, `onUnmount`, and
+`beforeCapture` to be `unknown` instead of `Error`. This more accurately matches behaviour of `componentDidCatch`, the
+lifecycle method the Sentry `ErrorBoundary` component uses.
```

Comment:
```suggestion
In v8, we are changing the `ErrorBoundary` error types returned from `onError`, `onReset`, `onUnmount`, and
`beforeCapture` to be `unknown` instead of `Error`. This more accurately matches the behavior of `componentDidCatch`, which is the
lifecycle method Sentry's `ErrorBoundary` component uses.
```